superset/assets/src/dashboard/util/publishSubscriberUtil.js

```diff
@@ -30,17 +32,76 @@ export function getUniqueActionsForSlice(slice) {
 export function getModalSliceIDFor(publishSubscriberMap, publisherId) {
     if (publishSubscriberMap && publishSubscriberMap.hasOwnProperty('subscribers')) {
         const subscribers = Object.values(publishSubscriberMap.subscribers);
-        var item = _.find(subscribers, function (item) {
+        var item = find(subscribers, function (item) {
             return ((item.actions.indexOf(USE_AS_MODAL) > -1) && (Object.keys(item.linked_slices).indexOf(publisherId) != -1));
         });
         return item ? item.id : undefined;
     }
     return undefined
 }
 
+export function getSubHeaderForSlice(subscribers, chartId, filters) {
+    let subHeader = '';
+    if (chartId != -1 && subscribers && subscribers[chartId]) {
+        const subscriber = subscribers[chartId];
+        if (subscriber.actions.indexOf(INCLUDE_IN_TITLE) > -1) {
+            subHeader = '';
```

Comment:
No need to initialised again subHeader. We already done in first line.

Reply:
Done